libs/examples/src/index.ts

```diff
@@ -203,6 +203,7 @@ import { DtExampleMicroChartInterpolated } from './micro-chart/micro-chart-inter
 import { DtExampleMicroChartStream } from './micro-chart/micro-chart-stream-example/micro-chart-stream-example';
 import { DtExampleOverlayComplexContent } from './overlay/overlay-complex-content-example/overlay-complex-content-example';
 import { DtExampleOverlayDefault } from './overlay/overlay-default-example/overlay-default-example';
+import { DtExampleOverlayFullscreen } from './overlay/overlay-fullscreen-example/overlay-fullscreen-example';
```

Comment:
NIT: I guess this fixes an issue in barista with the newly introduced full screen overlay?

Reply:
The file is generated every time one runs npm run demos. If it was not run when a new dome was created or not checked in at the time, it will be changed. 

I will remove it from this PR and create a separate one.